tests/test_install.sh

```diff
@@ -1,37 +1,39 @@
 #!/bin/bash
+set -ueo pipefail
 
-# Github Image Registry PAT
-if [ -z "${CODESEALER_TOKEN}" ]; then
+if [[ "$#" -ne 1 ]]; then
   echo "####################################################################################################################"
   echo "#"
-  echo "#  Please set CODESEALER_TOKEN variable"
-  echo "#"
-  echo "#    $ export CODESEALER_TOKEN=<Registry Access Token>"
+  echo "#  Usage: ./test_install.sh [install | uninstall | upgrade]"
   echo "#"
   echo "####################################################################################################################"
   exit 1
 fi
 
-set -ueo pipefail
+clear
 
-if [[ "$#" -ne 1 ]]; then
+# Github Image Registry PAT
+if [ -z "${CODESEALER_TOKEN}" ]; then
   echo "####################################################################################################################"
   echo "#"
-  echo "#  Usage: ./test_install.sh [install | uninstall | upgrade]"
+  echo "#  Please set CODESEALER_TOKEN variable"
   echo "#"
-  echo "####################################################################################################################"
-  exit 1
+  echo "#    $ export CODESEALER_TOKEN=<Registry Access Token>"
+  echo "#"
+  echo "#####################################################################################################################"
```

Comment:
These were swapped for a reason. Please revert.

Reply:
Done